models/issue_tracked_time.go

```diff
@@ -299,13 +303,11 @@ func deleteTime(e Engine, t *TrackedTime) error {
 
 // GetTrackedTimeByID returns raw TrackedTime without loading attributes by id
 func GetTrackedTimeByID(id int64) (*TrackedTime, error) {
-	time := &TrackedTime{
-		ID: id,
-	}
-	has, err := x.Get(time)
+	time := new(TrackedTime)
+	has, err := x.ID(id).Get(time)
 	if err != nil {
 		return nil, err
-	} else if !has {
+	} else if !has || time.Deleted {
```

Comment:
Maybe it's not a good idea to add this on this function.

Reply:
changed @lunny